src/Core/MailTemplates/Handlebars/OTPEmail.text.hbs

```diff
@@ -0,0 +1,5 @@
+﻿{{#>BasicTextLayout}}
+Your email verification code is: {{Token}}
+
+Use this code to complete the protected action in Bitwarden.
```

Comment:
I think the goal of using "protected action" was to simplify development time spent on the email. If it is easy enough to pass in a variable for the protected action (export vault, view api key, etc), let's do it. Otherwise "protected action" is acceptable.